embedded/tbtree/tbtree.go

```diff
@@ -646,13 +672,20 @@ func greatestKeyOfSize(size int) []byte {
 	return k
 }
 
+func minNodeSize(maxKeySize, maxValueSize int) int {
+	// space for at least two children is required for inner nodes
+	// 31 bytes are fixed in leafNode serialization while 29 bytes are fixed in innerNodes
```

Comment:
Is there a way to calculate this 31 instead of hardcoding that value or to put it closer to the serialization code?

It may lead to very unpleasant issues if we forget to update it when the btree structure changes again.

Reply:
I think it will be better to handle the changes with care. I added a direct reference into the size calculation functions for lead and inner nodes describing that this function needs to be revised if any change is made